apps/central-scan/frontend/src/screens/admin_actions_screen.tsx

```diff
@@ -249,13 +254,8 @@ export function AdminActionsScreen({
       )}
       {unconfigureFlowState === 'double-confirmation' && (
         <Modal
-          centerContent
-          content={
-            <Prose textCenter>
-              <h1>Are you sure?</h1>
-              <p>This can not be undone.</p>
-            </Prose>
-          }
+          title="Are you sure?"
+          content={<P>This can not be undone.</P>}
```

Comment:
i know this is copied over but i think grammar police would dictate that this be changed to "This **cannot** be undone."

Reply:
Ah, good catch - agreed, definitely worth rolling into this change